docs/_includes/install/universal.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/sh -e
 
+if [ -f "clickhouse" ]; then
+    rm --interactive=never clickhouse || { echo "clickhouse already exists and I'm not able to remove it :("; exit 1; }
```

Comment:
A smile should not be used in error messages because it adds an emotional load to a failure event, encouraging users to empathize and share these emotions.